### ISSUE TITLE
Two decimals

### ISSUE THREAD

**Issue by mortenvernegg** [getflourish/Sketch-Color-Contrast-Analyser#25]
Thanks for a great plugin!

Is it possible to show two decimals in the contrast ratio? 

I recently talked with an accessibility expert who had tested with users who were unable to read the text at contrast ratio 4.45, but managed it on 4.50. I also think some accessibility testing tools use two decimals.

**Comment by getflourish** [comment]
Thanks for the suggestion.

You can read more about why the current version has only one decimal here: 

https://github.com/getflourish/Sketch-Color-Contrast-Analyser/commit/52fe650df49c981aa6350635d4b50b76eb95e047 

That change removed any rounding, but the displayed value is capped to one decimal. 

I changes this here:
https://github.com/getflourish/Sketch-Color-Contrast-Analyser/commit/87afb115a1e698c4b393e3cf40f605169e9e724b

I might need to bump the version for auto updates.
